docs/en/getting-started/example-datasets/uk-price-paid.md

```diff
@@ -42,65 +33,68 @@ CREATE TABLE uk_price_paid
 - transforming `type` and `duration` to more readable Enum fields with function [transform](../../sql-reference/functions/other-functions.md#transform);
 - transforming `is_new` and `category` fields from single-character string (`Y`/`N` and `A`/`B`) to [UInt8](../../sql-reference/data-types/int-uint.md#uint8-uint16-uint32-uint64-uint256-int8-int16-int32-int64-int128-int256) field with 0 and 1.
 
-Preprocessed data is piped directly to `clickhouse-client` to be inserted into ClickHouse table in streaming fashion.
-
 ```bash
-clickhouse-local --input-format CSV --structure '
-    uuid String,
-    price UInt32,
-    time DateTime,
-    postcode String,
-    a String,
-    b String,
-    c String,
-    addr1 String,
-    addr2 String,
-    street String,
-    locality String,
-    town String,
-    district String,
-    county String,
-    d String,
-    e String
-' --query "
-    WITH splitByChar(' ', postcode) AS p
-    SELECT
-        price,
-        toDate(time) AS date,
-        p[1] AS postcode1,
-        p[2] AS postcode2,
-        transform(a, ['T', 'S', 'D', 'F', 'O'], ['terraced', 'semi-detached', 'detached', 'flat', 'other']) AS type,
-        b = 'Y' AS is_new,
-        transform(c, ['F', 'L', 'U'], ['freehold', 'leasehold', 'unknown']) AS duration,
-        addr1,
-        addr2,
-        street,
-        locality,
-        town,
-        district,
-        county,
-        d = 'B' AS category
-    FROM table" --date_time_input_format best_effort < pp-complete.csv | clickhouse-client --query "INSERT INTO uk_price_paid FORMAT TSV"
+INSERT INTO uk_price_paid
+WITH 
+    splitByChar(' ', postcode) AS p
+SELECT
+    replaceRegexpAll(uuid_string, '{|}','') AS uuid,
+    toUInt32(price_string) AS price,
+    parseDateTimeBestEffortUS(time) AS date,
+    p[1] AS postcode1,
+    p[2] AS postcode2,
+    transform(a, ['T', 'S', 'D', 'F', 'O'], ['terraced', 'semi-detached', 'detached', 'flat', 'other']) AS type,
+    b = 'Y' AS is_new,
+    transform(c, ['F', 'L', 'U'], ['freehold', 'leasehold', 'unknown']) AS duration,
+    addr1,
+    addr2,
+    street,
+    locality,
+    town,
+    district,
+    county,
+    d = 'B' AS category,
+    e = 'B' AS category2
+FROM url(
+	'http://prod.publicdata.landregistry.gov.uk.s3-website-eu-west-1.amazonaws.com/pp-complete.csv', 
+    'CSV',
+    'uuid_string String,
+     price_string String,
+     time String,
+     postcode String,
+     a String,
+     b String,
+     c String,
+     addr1 String,
+     addr2 String,
+     street String,
+     locality String,
+     town String,
+     district String,
+     county String,
+     d String,
+     e String'
+) 
+SETTINGS max_http_get_redirects=1;
 ```
 
-It will take about 40 seconds.
+It will take about 2 minutes depending on where you are in the world, and where your ClickHouse servers are. Almost all of the time is the download time of the CSV file from the UK government server.
```

Comment:
I don't like these changes for the following reasons:
- we cannot longer highlight the data import speed: it was 40 seconds but now it is two minutes, and it is completely up to how the UK government serves it to us.

The query about `total_bytes` does not give you twice the data volume for the replicated table.  This example is simply wrong.